bots/machine/testvm_core.py

```diff
@@ -0,0 +1,557 @@
+#!/usr/bin/python3 -u
```

Comment:
Please don't put a shebang here. testvm.py is executable (on purpose, it has a `__main__()`),  but this library is not.